keyboards/honeycomb/rules.mk

```
@@ -0,0 +1,82 @@
+
+OPT_DEFS += -Dhoneycomb_PROMICRO
```

This should all be upper case.

```suggestion
OPT_DEFS += -DHONEYCOMB_PROMICRO
```